app/src/pages/inside/common/filterEntitiesGroups/stepLevelEntities.jsx

```diff
@@ -675,50 +668,11 @@ export class StepLevelEntities extends Component {
 
   bindDefaultValue = bindDefaultValue;
 
-  handleAdd = (entity, ...rest) => {
-    if (entity.id === ENTITY_BTS_ISSUES && entity.value.value === 'FALSE') {
-      this.props.onFilterAdd(
-        [
-          entity,
-          {
-            id: ENTITY_ISSUE_ID,
-            filteringField: ENTITY_ISSUE_ID,
-            value: {
-              condition: CONDITION_EX,
-              value: 'TRUE',
-            },
-          },
-        ],
-        ...rest,
-      );
-    } else {
-      this.props.onFilterAdd(entity, ...rest);
-    }
-  };
+  handleAdd = (entity, ...rest) => this.props.onFilterAdd(entity, ...rest);
```

Comment:
It looks like these methods are no longer needed and should be replaced with the value from props in the component's render method.